test/test/com/github/tombentley/alabama/canonicalExample.ceylon

```diff
@@ -32,4 +34,17 @@ shared void canonicalExample() {
     };
     Person p2 = deserialize<Person>(json);
     print(json);
+}
+
+test shared void itShouldSupportClassesWithUnionTypeFields() {
```

Comment:
Is *field* really the right word to use in the method name, wouldn't *attribute* be more appropriate?